src/Illuminate/Collections/Traits/EnumeratesValues.php

```diff
@@ -342,7 +342,7 @@ public function value($key, $default = null)
      *
      * @template TEnsureOfType
      *
-     * @param  class-string<TEnsureOfType>|array<array-key, class-string<TEnsureOfType>>  $type
+     * @param  class-string<TEnsureOfType>|array<array-key, class-string<TEnsureOfType>>|'string'|'int'|'float'|'bool'|'array'|'null'  $type
```

Comment:
Might be shortened to
```suggestion
     * @param  class-string<TEnsureOfType>|array<array-key, class-string<TEnsureOfType>>|scalar|'array'|'null'  $type
```

Reply:
I believe this change was wrong, see https://github.com/laravel/framework/pull/54919 for an explanation